mod/events.php

```diff
@@ -9,8 +9,9 @@
 require_once('include/items.php');
 
 function events_init(&$a) {
-	if(! local_user())
+	if (! local_user()) {
 		return;
+	}
 
 	if($a->argc == 1) {
```

Comment:
Not sure I get it. Can you see line 16 of `mod/events.php` the `if` conditional with a missing space after the `if`?